### PR TITLE
chore: one-time Sourcery clean-up

### PR DESCRIPTION
**Pull request by BoboTiG**
Hello,

If you are interested in such changes, LMK if you also have adjustments to make :)

### CODE REVIEW

File: medusa/_cli/analyse.py [hunk 1]
```diff
@@ -31,9 +31,7 @@ def main():
         output_dict = analyse(vyper_ast)
         formatted_analysis = format_analysis(output_dict)
 
-        # Write to output file
-        output_file = args["--output"]
-        if output_file:
+        if output_file := args["--output"]:
```
Comment (BoboTiG) [inline medusa/_cli/analyse.py:35]:
```suggestion
        # Write to output file
        if output_file := args["--output"]:
```

**Comment by tserg** [comment]
Looks good, can you resolve the linting conflicts with `black`?

**Comment by BoboTiG** [comment]
Done ✔️

**Review by tserg — APPROVED** [review]
LGTM, thanks!

**Pull request merged**